internal/character/clara/skill.go

```diff
@@ -0,0 +1,26 @@
+package clara
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+func (c *char) Skill(target key.TargetID, state info.ActionState) {
+	c.engine.Attack(info.Attack{
+		Source:     c.id,
+		Targets:    c.engine.Enemies(),
+		DamageType: model.DamageType_PHYSICAL,
+		AttackType: model.AttackType_SKILL,
+		BaseDamage: info.DamageMap{
+			model.DamageFormula_BY_ATK: skill[c.info.SkillLevelIndex()],
+		},
+		StanceDamage: 30.0,
+		EnergyGain:   30,
+	})
+
+	// E1 check
+	c.e1()
```

Comment:
nit: would rather inline the logic here so it is more clear that E0 is counter mark removal